src/export.ts

```diff
@@ -0,0 +1,7 @@
+export { PeripheralOptions } from './views/nodes/peripheralnode';
```

Comment:
As mentioned, perhaps extract these all into a common api file?

Reply:
I moved them to `api.ts` file.